app/components/projects-carousel.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+    Carousel,
+    CarouselContent,
+    CarouselDots,
+    CarouselItem,
+    CarouselNext,
+    CarouselPrevious,
+} from "../components/ui/carousel"
+import Project from "./project"
+
+interface ProjectsCarouselProps {
+    projects: { image?: string; title?: string; subtitle?: string; description?: string; link?: string; }[];
```

Comment:
can we reuse the ProjectProps here?
```suggestion
    projects: ProjectProps[];
```